src/hotspot/os/linux/os_linux.cpp

```diff
@@ -3619,7 +3625,7 @@ static void set_coredump_filter(CoredumpFilterBit bit) {
 
 static size_t _large_page_size = 0;
 
-static size_t scan_default_large_page_size() {
+size_t os::Linux::scan_default_large_page_size() {
```

Comment:
I think this is not needed anymore, or? You can revert this back to a local static function, and call that in `os::large_page_init()`

Reply:
Fixed this.